app/containers/GameBoard.jsx

```diff
@@ -86,13 +112,20 @@ class GameBoard extends Component {
   // checks if more than one player has the highest card
   compareWinnerAndDuplicates(array, valueOfDuplicates, currCards) {
     const { decks, cardsToBeWon } = this.state;
+
     const totalWin = [];
 
     if (array[1] > valueOfDuplicates) {
-      currCards.forEach(card => {
+      currCards.forEach(card => {  
+        // here you are mutating state
+        // This will not trigger a re-render in react
+        // since you are not calling setState
+        // this can lead to confusing behavior in your component
+        // copy decks. change the part you want, call setState to update the decks
```

Comment:
I see what you're saying. Line 88 is a reference to values in state, not a copy.